scripts/install

```diff
@@ -26,6 +26,7 @@ path_url=$YNH_APP_ARG_PATH
 admin=$YNH_APP_ARG_ADMIN
 query_logging=$YNH_APP_ARG_QUERY_LOGGING
 enable_dhcp=$YNH_APP_ARG_ENABLE_DHCP
+is_public=$YNH_APP_ARG_IS_PUBLIC
```

Comment:
Why would you have a public Pi-Hole ?

Reply:
Is for me is better need to see that is option but need login page from pi hole look
https://51.68.44.69/admin/index.php
For me need public acces for that but need login page

Reply:
Indeed, Pi-Hole does not have a login page. But why should a Pi-Hole be public ?
It's an admin tools, for your local network. You should be the only one to see it.

Reply:
Pi-Hole have login page https://51.68.44.69/admin/index.php?login !
And this demand is for my enterprise.

Reply:
I had them with the installation of pi-hole on my debian stretch
With a script install

Reply:
I can't find any documentation about it... Except how to reset the password.
If you find anything about it, and if it works with the version in use currently. We can implement it.